generators/docker-compose/templates/realm-config/jhipster-realm.json.ejs

```diff
@@ -2492,12 +2492,70 @@
     "clientOfflineSessionIdleTimeout": "0",
     "cibaInterval": "5"
   },
-  "keycloakVersion": "16.1.0",
+  "keycloakVersion": "18.0.0",
   "userManagedAccessAllowed": false,
   "clientProfiles": {
     "profiles": []
   },
   "clientPolicies": {
     "policies": []
-  }
+  },
+  "users": [
```

Comment:
any specific reason to combine users data alongside realm?

Reply:
@vishal423 splitting not allowed anymore starting from keylcoak 17